.github/workflows/build-artifact.yaml

```diff
@@ -76,6 +76,17 @@ jobs:
       - build
       - in-service
     steps:
+      - name: Verify ccache availability
+        shell: bash
+        run: |
+          if [ ! -d "/mnt/MLPerf/ccache" ]; then
+            echo "::error::NFS drive is not mounted; build machine not properly provisioned."
```

Comment:
I would recommend putting in a `title=` https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-error-message with lowercase ASCII alphabet characters with only `-` as the delimiter.

For ex. `::error title=ccache-mlperf-not-mounted:: NFS drive is not mounted; build machine not properly provisioned.`

Reply:
BTW for context - this is so that we can easily grab error messages later programmatically via the API when analyzing CI data

Reply:
I'm all for that!  I'm still learning how to make GitHub usable for CI.  It seems to have some useful features that are well hidden.  I'll take all the lemonade I can get.